Codeminer42/cm42-central
# Unstarded story state

# Set an estimated story to unstarted automatically

### Feature
Set an estimable story as unstarted if it gets estimated and set it back to unscheduled if it gets unestimated.

An estimable story can't have a state of unstarted while it's not estimated and can't have a state of unscheduled while it's estimated.

### Q & A

#### Estimable story types?
So far only the `feature` stories are estimable.

#### How to test?
In order to test you should check if when you estimate an estimable story (feature one) it will change the state to `unstarted` and when unestimate it will set the state back to `unscheduled`. Make sure the stories the are not estimable (chore, bug and release ones) don't have the same behavior.
